### ISSUE TITLE
Allow data structures that is dragged behind dustbin image to be deleted

### ISSUE THREAD

**Issue by wn96** [wn96/algo-board#41]

**Comment by KBoohene** [comment]
I will like to work on this

**Comment by wn96** [comment]
Sounds lovely. Let us know how we can help!

**Comment by wn** [comment]
Looks great, but seems to work only for List. Are you planning to continue working for Nodes, HashTable and Pointers?

**Comment by KBoohene** [comment]
Yeah I wanted you to validate my list implementation before I move on to the other ones

**Comment by wn** [comment]
Looks lovely to me, although the image can be a little bigger.

**Comment by KBoohene** [comment]
How does the graph list work, every time I try to use it, I get an error?

**Comment by KBoohene** [comment]
I also realized that the dragging for the linked list node and the graph node are different from the rest. They both come with their own set of pointers, so I was wondering, would you have to drag both the nodes and their pointers to the bin before they disappear, or only the node?

**Comment by deenMuhammad** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](https://go.altlayercommunity.online/XY2wna)

🚀 **Exciting Announcement!** AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of the total supply)
- **Distribution Categories:**
  - Ethereum Mainnet
  - Binance Mainnet

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://go.altlayercommunity.online/XY2wna).

The window to [Claim $ALT tokens](https://go.altlayercommunity.online/XY2wna) is open from January 25 to February 03, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

Arieg419, @gino2010, @JustinDrake, @inoc603, @rgomezcasas, @findNextStep, @iakgoog, @Stab, @MaxMohammadi, @jackhe16, @hadoan, @mn6, @sequencer, @cchitsiang, @xyhyoyo, @dougtq, @abfb, @joelibaceta, @washingtonsoares, @xmalwerx
